client/reader/style.scss

```diff
@@ -48,16 +103,6 @@ body.is-section-reader {
 	margin-top: 0;
 }
 
-.is-reader-page .recommended-for-you.main,
-.is-reader-page .following.main,
-.is-reader-page .tag-stream__main.main,
-.is-reader-page .search-stream__with-sites.main {
-	margin: 0 auto;
-	@include breakpoint-deprecated( "<660px" ) {
-		margin: 16px auto;
-	}
-}
-
```

Comment:
Using common styles which is getting applied across all reader screens (almost) so removed this.